torch/distributed/fsdp/flat_param.py

```diff
@@ -304,6 +304,11 @@ def __init__(
         self._use_orig_params = use_orig_params
         self._training_state = HandleTrainingState.IDLE
         self._debug_level = dist.get_debug_level()
+        self._computation_stream = (
```

Comment:
For some `FlattenParamsWrapper` tests, there is no CUDA available. In `FullyShardedDataParallel`, we require `torch.cuda.is_available()`, so this is `if` / `else` is only relevant for unit testing.